crates/editor/src/editor.rs

```diff
@@ -4493,15 +4493,31 @@ impl Editor {
                 continue;
             }
 
-            // If the selection is empty and the cursor is in the leading whitespace before the
-            // suggested indentation, then auto-indent the line.
             let cursor = selection.head();
             let current_indent = snapshot.indent_size_for_line(cursor.row);
+
+            let can_accept_suggestion = self.selections.count() == 1
+                && cursor.column >= current_indent.len
+                && self.has_active_inline_completion(cx);
+
+            let active_suggestion_indent = if can_accept_suggestion {
```

Comment:
If this check is not done, we get another issue where
```
fn foo() {
••ˇ• // assume a suggestion exists with proper 4 space indent
}
```
will show a completion but not auto-indent. The problematic behavior is as follows:
1. After first tab
```
fn foo() {
••••ˇ• // editor still shows completion
}
```
2. Second tab will not accept shown suggestion and will instead tab again
```
fn foo() {
••••••••ˇ•
}
```